src/screepsServer.js

```diff
@@ -142,7 +145,7 @@ class ScreepsServer extends EventEmitter {
         Start processes and connect driver.
     */
     async start() {
-        this.emit('info', `Server version ${require('screeps').version}`);
+        // this.emit('info', `Server version ${require('screeps').version}`);
```

Comment:
This is the only point we actually reference @screeps/screeps directly. To read the combined server version.

Reply:
I think this could be removed